pkg/controller/podautoscaler/horizontal.go

```diff
@@ -48,11 +48,12 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 )
 
-var (
-	// Usage shoud exceed the tolerance before we start downscale or upscale the pods.
-	// TODO: make it a flag or HPA spec element.
-	tolerance = 0.1
+const (
+	// DefaultTolerance used to calculating when to scale up/scale down.
+	DefaultTolerance = 0.1
```

Comment:
Move this to replica_calculator.go, since we actually use it there.

Reply:
Yup, good point!